### PR TITLE
Summary Snapshot Module

### PR DESCRIPTION
**Pull request by andrewelamb**

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -114,4 +114,17 @@ function(input, output) {
     )
   )
 
+  module_server(
+    id = "projectlive_summary_snapshot_module",
+    server_function_name = "summary_snapshot_module_server",
+    ui_function_name = "summary_snapshot_module_ui",
+    example_module_ids = c("example_1"),
+    example_server_files = c(
+      "inst/projectlive_examples/summary_snapshot_module_server_1.R"
```
Comment (milen-sage) [inline server.R:123]: @andrewelamb just curious, any specific reason for the _1 in the filenames?

File: inst/projectlive_json/csbc_summary_snapshot_module.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (milen-sage) [inline inst/projectlive_json/csbc_summary_snapshot_module.json:1]: The projectLive_json/ contains examples of specific project configurations for different modules? I haven't looked through the folder, yet; does it contain a generic config using example data agnostic to specific project? That might be useful to substitute project-specific files.

**Review by milen-sage — APPROVED** [review]
It's nice how lightweight the shiny logic is and that the config files contain the bulk. 

Trying to understand a little bit better how the code works: are the rds files generated periodically (e.g. cron) or using a different approach?

Unrelated, are there docs on what the config parameters do and what the different options are (e.g. tooltips has 'count' and 'fill' do these correspond to a specific R function parameters; are there other options)?

I also left some questions in line, but this looks good for now!

File: server.R (hunk 1 above)
Comment (andrewelamb) [reply server.R:123]: Exactly!

File: inst/projectlive_json/csbc_summary_snapshot_module.json (hunk 2 above)
Comment (andrewelamb) [reply inst/projectlive_json/csbc_summary_snapshot_module.json:1]: Yeah right now they include examples from the CSBC app. I do agree some non-specific examples would be great, they could be used for unit testing in the projectlive.modules repo.

**Comment by andrewelamb** [comment]
@milen-sage 

"Trying to understand a little bit better how the code works: are the rds files generated periodically (e.g. cron) or using a different approach?"

It depends. For NF we do now have a cronjob that generates them once a day. For CSBC they were done manually since it was just for a demo.

"Unrelated, are there docs on what the config parameters do and what the different options are (e.g. tooltips has 'count' and 'fill' do these correspond to a specific R function parameters; are there other options)?"

Not at the moment since I was the main "user". I would like to do this in the future.

File: inst/projectlive_json/csbc_summary_snapshot_module.json (hunk 2 above)
Comment (milen-sage) [reply inst/projectlive_json/csbc_summary_snapshot_module.json:1]: Could you start a gh issue for the generic example? We can triage for some of the next sprints.

**Comment by milen-sage** [comment]
> Not at the moment since I was the main "user". I would like to do this in the future.

Could you start a gh issue for the docs? We can triage for a sprint as well.

**Comment by milen-sage** [comment]
> It depends. For NF we do now have a cronjob that generates them once a day. For CSBC they were done manually since it was just for a demo.

What would be a good approach to handle this generally? Is there any disadvantage to a scheduled job other than having to deploy the job somewhere?

**Comment by andrewelamb** [comment]
That's a good question. After getting onboarded to FAIR, it seems like it would be out of our scope, and should be left up to the managers of the project to handle, likely with the help of DPE. We could point them to what [NF is doing](https://github.com/nf-osi/jobs/tree/main/projectlive-nf-datafiles). I think the only reason to not have a scheduled job besides the upfront work, is if you want to control what goes into the app, ie. you want to have something like releases.

**Comment by milen-sage** [comment]
> likely with the help of DPE.

Agreed 100% - this is in the wheelhouse of DPE. There are a few other items that fall in this category and we can discuss with Tom and Sophia, this more generic type of hand off (i.e. setting up periodic compute jobs/workflows).

@ychae let's sync up after home week, on how we want to get the ball rolling on hand-offs with DPE.

**Pull request merged**